setup.py

```diff
@@ -8,7 +8,7 @@
     "click",
     "cookiecutter",
     "matplotlib",
-    "mesa_viz_tornado @ git+https://github.com/rht/mesa-viz-tornado@a4d9242e90ef7f1fcd388fb5c6d72a4177a76bdd",
+    "mesa_viz_tornado,
```

Comment:
Syntax error.

Reply:
Sorry about that, that's what I get for being lazy and seeing what it was like to edit the file in the browser
 now fixed